src/bot/integrations/discord.ts

```diff
@@ -371,6 +386,51 @@ class Discord extends Integration {
     } catch (e) {
       warning(e.stack);
     }
+
+    try{
+      this.changeClientOnlinePresence();
+    }catch (e){
+      warning(e.stack);
+    }
+  }
+
+  @onChange('onlinePresence')
+  async changeClientOnlinePresence() {
+    try{
+      if (api.isStreamOnline) {
+        this.client?.user?.setStatus('online')
```

Comment:
please use await instead of then/catch

you can catch await errors with try/catch, this will minify code a lot